### PR TITLE
Add the list/describe commands

### PR DESCRIPTION
**Pull request by cdavid**
Based on feedback, we need to distinguish between
cases when the user wants to list the objects
and when the user wants to describe a single object.

### CODE REVIEW

File: cmd/backup/backup.go [hunk 1]
```diff
@@ -38,8 +38,18 @@ var BackupCmd = &cobra.Command{
 
 var getBackupCmd = &cobra.Command{
 	Use:   "get",
-	Short: "Get list of existing backups available for a cluster in YugabyteDB Managed",
-	Long:  "Get list of existing backups available for a cluster in YugabyteDB Managed",
+	Short: "Get backup details in YugabyteDB Managed",
```
Comment (shinji62) [inline cmd/backup/backup.go:41]: The cli is still quite new, should just remove the command fully ?

Comment (posriniv) [reply cmd/backup/backup.go:41]: Why should we remove the command when it is working?

**Review by kc-yb — COMMENTED** [review]
@cdavid Why not have list and describe? get does not intuitively imply listing

**Review by posriniv — APPROVED** [review]
Looks okay to me!

**Comment by cdavid** [comment]
> @cdavid Why not have list and describe? get does not intuitively imply listing

That is what I am adding here. I am adding list and describe, while maintaining backwards compatibility for the existing get command until it is deprecated in e.g.: next release.

File: cmd/backup/backup.go (hunk 1 above)
Comment (cdavid) [reply cmd/backup/backup.go:41]: We will properly run through a deprecation cycle. We will remove it with the next release.

**Pull request merged**